### PR TITLE
Logback integration

### PR DESCRIPTION
**Pull request by fredex42**
Removing log4j and using logback/slf4j instead - this allows all of the different components to use the same underlying logging framework and therefore get a complete logfile. Making debugging much easier...

### CODE REVIEW

**Comment by fredex42** [comment]
@tomrf1 worked out how to do it :) If you have a minute could you review? thanks

File: build.sbt [hunk 1]
```diff
@@ -19,20 +19,21 @@ libraryDependencies += "com.typesafe.akka" %% "akka-http" % "10.0.10"
...
 libraryDependencies ++= Seq(
-  "org.apache.logging.log4j" % "log4j-api" % "2.9.1",
-  "org.apache.logging.log4j" % "log4j-core" % "2.9.1",
-  "org.apache.logging.log4j" %% "log4j-api-scala" % "11.0",
-  "com.fasterxml.jackson.dataformat" % "jackson-dataformat-yaml" % "2.7.3",
-  "com.fasterxml.jackson.core" % "jackson-databind" % "2.5.4",
-  "org.apache.logging.log4j" % "log4j-slf4j-impl" % "2.9.1"
+  // https://mvnrepository.com/artifact/ch.qos.logback/logback-classic
+  "ch.qos.logback" % "logback-classic" % "1.2.3",
+  "ch.qos.logback" % "logback-classic" % "1.2.3" % Test,
```
Comment (tomrf1) [inline build.sbt]: I think the line above makes this line redundant?

**Comment by tomrf1** [comment]
👍 glad you got this working!

File: build.sbt (hunk 1 above)
Comment (fredex42) [reply build.sbt]: sorted

**Pull request merged**